### PR TITLE
Added GPIO init, in a very awful but functional way

### PR DESCRIPTION
**Pull request by mfruge9214**

### CODE REVIEW

File: src/motor_control.c [hunk 1]
```diff
@@ -134,17 +138,35 @@ void motor_change_pwm_duty_cycle(Motor_t motor, uint8_t duty_cycle) {
...
-//    MAP_GPIODirModeSet(X_MOTOR_GPIO_BASE, X_NFAULT, GPIO_DIR_MODE_IN);
-
-//    MAP_GPIOPadConfigSet(); // used to set drive strength
+    MAP_SysCtlPeripheralEnable(motor.M0.base);
+    MAP_SysCtlPeripheralEnable(motor.M1.base);
+    MAP_SysCtlPeripheralEnable(motor.M1.base);
+    MAP_SysCtlPeripheralEnable(motor.M1.base);
+    MAP_SysCtlPeripheralEnable(motor.M1.base);
+    MAP_SysCtlPeripheralEnable(motor.M1.base);
+    MAP_SysCtlPeripheralEnable(motor.M1.base);
```
Comment (jackdski) [inline src/motor_control.c]: Why is this line repeated?

File: src/motor_control.c [hunk 2]
```diff
@@ -134,17 +138,35 @@ void motor_change_pwm_duty_cycle(Motor_t motor, uint8_t duty_cycle) {
  *  DECAY - ?
  *  TRQ - ?
  */
-void motor_init_gpio(void) {
+void motor_init_gpio(Motor_t motor) {
+
+    // TODO: This is very hardcoded and redundant, would like to discuss a better way of doing this
+
+        // After doing a search, I found there is no safe way to iterate through members of a struct
```
Comment (jackdski) [inline src/motor_control.c]: Remove these lines and commit so this doesn't show up on the master branch

**Review by mfruge9214 — COMMENTED** [review]
Fixed comments, ready to merge

**Review by jackdski — APPROVED** [review]
Looks good

**Pull request merged**